crates/pallets/starknet/src/tests/invoke_tx.rs

```diff
@@ -66,7 +66,14 @@ fn given_hardcoded_contract_run_invoke_tx_then_it_works() {
     new_test_ext::<MockRuntime>().execute_with(|| {
         basic_test_setup(2);
 
-        let transaction = get_invoke_dummy(Starknet::chain_id(), NONCE_ZERO);
+        let mut transaction = get_invoke_dummy_v0(Starknet::chain_id());
```

Comment:
You modified an existing test, that is now failing.
put back `given_hardcoded_contract_run_invoke_tx_then_it_works` as it was. It was working.

And add your tx_v0 test as another distinct test